api/query_builder.go

```diff
@@ -298,6 +299,12 @@ var PullRequestFields = append(IssueFields,
 	"statusCheckRollup",
 )
 
+// Some fields are only valid in the context of issues.
+var IssueOnlyFields = []string{
```

Comment:
I don't think this needs to be exported but otherwise I'm happy with the change.

Reply:
good catch, will make private

Reply:
implemented in 9823fcc93c0128bb946de5c604267ea7789d844e